crates/solver/src/settlement.rs

```diff
@@ -330,7 +330,6 @@ impl Settlement {
     // Checks whether the settlement prices do not deviate more than max_settlement_price_deviation from the auction prices on certain pairs.
     pub fn satisfies_price_checks(
         &self,
-        auction_id: u64,
```

Comment:
Only used for logging, but we are in an `info_span` that includes this information so I removed it.